### DISCUSSION TITLE
Hauptbildschirm

### DISCUSSION THREAD

**Discussion by Achim824** [softwarecrash/Victron2MQTT#32] [Q&A]
Hallo,
bin neu hier und habe eine Frage zum Victron2MQTT.
Habe ein Victron2MQTT gekauft und über Wlan eingerichtet.
In der Web Oberfläche im Hauptbildschirm wird aber nur "Einstellung" und Fernbedienung angezeigt aber keine Daten vom Mppt.
Was könnte die Ursache sein?
Kann mir jemand Helfen.
Danke

**Comment by softwarecrash** [comment]
hallo,
was gibt den esp-ip/livejson aus?

**Comment by Achim824** [comment]
hallo,Vielen Dank,  musste den Schalter auf der Platine umstellen. Kann ich z.bden Wert Bulk noch anzeigen.Danke Am 11.12.2024 18:17 schrieb Rocky ***@***.***>:
hallo,
was gibt den /livejson aus?

**Reply by softwarecrash** [reply]
aktuell leider nicht

**Comment by Achim824** [comment]
Danke Am 13.12.2024 11:52 schrieb Rocky ***@***.***>:
aktuell leider nicht
